src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -456,6 +462,7 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
   private forEachObserver(observerFunc: (observer: DeviceChangeObserver) => void): void {
     for (const observer of this.deviceChangeObservers) {
       new AsyncScheduler().start(() => {
+        /* istanbul ignore else */
```

Comment:
what is "istanbul ignore else " ?

Reply:
Istanbul will skip the `else` case for coverage. Unless we make `forEachObserver` public, it's impossible to cover the `else` case.

SDK is iterating listeners in `this.deviceChangeObservers`. In the next tick (`setTimeout` with zero delay), SDK checks whether the listener exists in `this.deviceChangeObservers`.